CMakeLists.txt

```diff
@@ -439,6 +439,10 @@ set(SOUND_FILES
     ${PROJECT_SOURCE_DIR}/src/sound/system.c
 )
 set(WIDGET_FILES
+    ${PROJECT_SOURCE_DIR}/src/widget/sidebar/city.c
+    ${PROJECT_SOURCE_DIR}/src/widget/sidebar/common.c
+    ${PROJECT_SOURCE_DIR}/src/widget/sidebar/editor.c
+    ${PROJECT_SOURCE_DIR}/src/widget/sidebar/extra.c
```

Comment:
Put them at the bottom of the WIDGET_FILES instead: main dir comes before subdirs. Same goes for the `#include` statements in the .c files.